riotfile.py

```diff
@@ -1155,6 +1155,39 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
                 ),
             ],
         ),
+        Venv(
+            name="pymysql",
+            command="pytest {cmdargs} tests/contrib/pymysql",
+            venvs=[
+                Venv(
+                    pys="2.7",
+                    pkgs={
+                        "pymysql": [
+                            "~=0.7",
+                            "~=0.8",
+                        ],
+                    },
+                ),
+                Venv(
+                    pys="2.7",
+                    pkgs={
+                        "pymysql": [
+                            "~=0.9",
+                        ],
+                    },
+                ),
```

Comment:
```suggestion
                Venv(
                    pys=select_pys(),
                    pkgs={
                        "pymysql": [
                            "~=0.7",
                            "~=0.8",
                            "~=0.9",
                        ],
                    },
                ),
```

This seems more in-line with what the original tox was testing against?

Every version of Python * pymysql versions 0.7.x, 0.8.x, and 0.9.x.